docs/getting-started.md

```diff
@@ -1,58 +1,76 @@
-  - [Setting up Virtual Robocup 2021](#setting-up-virtual-robocup-2021)
-    - [Setting up the SSL Simulation Environment](#setting-up-the-ssl-simulation-environment)
-    - [Pushing a Dockerfile to dockerhub](#pushing-a-dockerfile-to-dockerhub)
+* [Software Setup](#software-setup)
+   * [Introduction](#introduction)
+   * [Installation and Setup](#installation-and-setup)
+      * [Operating Systems](#operating-systems)
+      * [Getting the Code](#getting-the-code)
+      * [Running the setup scripts](#running-the-setup-scripts)
+         * [Installing Software Dependencies](#installing-software-dependencies)
+         * [Installing an IDE](#installing-an-ide)
+            * [CLion](#clion)
+               * [Getting your Student License](#getting-your-student-license)
+               * [Installing CLion](#installing-clion)
+            * [VSCode](#vscode)
+   * [Building and Running the Code](#building-and-running-the-code)
+      * [Building from the command-line](#building-from-the-command-line)
+         * [Using the fuzzy finder](#using-the-fuzzy-finder)
+      * [Editing with Vim or NeoVim](#editing-with-vim-or-neovim)
+      * [Building with CLion](#building-with-clion)
+      * [With VSCode](#with-vscode)
+      * [Running our AI, Simulator, SimulatedTests or Robot Diagnostics](#running-our-ai-simulator-simulatedtests-or-robot-diagnostics)
+   * [Debugging](#debugging)
+      * [Debugging with CLion](#debugging-with-clion)
+      * [Debugging from the Command line](#debugging-from-the-command-line)
+   * [Profiling](#profiling)
+   * [Building for Jetson Nano](#building-for-jetson-nano)
+   * [Deploying to Jetson Nano](#deploying-to-jetson-nano)
+   * [Setting up Virtual Robocup 2021](#setting-up-virtual-robocup-2021)
+      * [Setting up the SSL Simulation Environment](#setting-up-the-ssl-simulation-environment)
+      * [Pushing a Dockerfile to dockerhub](#pushing-a-dockerfile-to-dockerhub)
+* [Workflow](#workflow)
+   * [Issue and Project Tracking](#issue-and-project-tracking)
+      * [Issues](#issues)
+   * [Git Workflow](#git-workflow)
+      * [Forking and Branching](#forking-and-branching)
+      * [Creating a new Branch](#creating-a-new-branch)
+         * [Why should you only create branches from "upstream/master"?](#why-should-you-only-create-branches-from-upstreammaster)
+      * [Making Commits](#making-commits)
+      * [Updating Your Branch and Resolving Conflicts](#updating-your-branch-and-resolving-conflicts)
+      * [Formatting Your Code](#formatting-your-code)
+      * [Pull Requests](#pull-requests)
+      * [Reviewing Pull Requests](#reviewing-pull-requests)
+   * [Example Workflow](#example-workflow)
+   * [Testing](#testing)
+
+# Software Setup
 
 ## Introduction
+<details>
+
 These instructions assume that you have the following accounts setup:
 - [Github](https://github.com/login)
-- [Slack](https://thunderbots.slack.com/)
+- [Discord](https://discord.gg/dDHYyX5uJv)
```

Comment:
you may or may not get randos joining the server

Reply:
I agree with Jon, I don't think we should have the link public, since unlike Slack, anyone can join.

Reply:
removed permalink